CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
-### 4.1.0 - 2018-11-06
+### 4.1.0 - 2018-11-19
 ### Added
 - Added more flexibility for getting a record's index
 
+### 4.0.18 - 2018-11-19
+### Added
+- Delete empty field and site groups on import with force
```

Comment:
This probably needs an extra newline after this line

Reply:
ok